net/ndproxy/src/opnsense/service/conf/actions.d/actions_ndproxy.conf

```diff
@@ -0,0 +1,24 @@
+[start]
+command:service ndproxy start
+parameters:
+type:script
+message:Starting ndproxy service
+
+[stop]
+command:service ndproxy stop
+parameters:
+type:script
+message:Stopping ndproxy service
+
+[restart]
+command:service ndproxy restart
+parameters:
+type:script
+message:Restarting ndproxy service
+description:Restart ndproxy service
+
+[status]
+command:/usr/local/sbin/pluginctl -s ndproxy status
```

Comment:
why pluginctl here? service ndproxy status not reliable?

Reply:
There is no status for ndproxy.

Reply:
root@opn-ce-01:/src/git/opnsense-plugins/net/ndproxy # service ndproxy status
/usr/local/etc/rc.d/ndproxy: unknown directive 'status'.
Usage: /usr/local/etc/rc.d/ndproxy [fast|force|one|quiet](start|stop|restart|rcvar|enable|disable|delete|enabled|describe|extracommands)

Reply:
alright, thanks